src/commons.ts

```diff
@@ -82,7 +82,7 @@ export default class Commons {
   }
 
   public async InitializeAutoUpload() {
-    const ignored = await AutoUploadService.GetIgnoredItems(

```

Comment:
Forgive me for not knowing Typescript, but this removal is allowing `GetIgnoredItems()` to execute asynchronously, correct?  May I ask what the reason for this change was and why it was decided to put the `await`s in the methods calling `InitializeAutoUpload` instead?

Reply:
Yes, that is correct. The handler functions just make sure that it is initialized before calling any of its methods. The async is used in the methods just wait for it to be initialled before calling.

Reply:
I removed await because `GetIgnoredItems()` is a synchronous function. Await is needed for `GetCustomSettings()` because it's async.